src/app/give-help/give-help.component.ts

```diff
@@ -1,11 +1,19 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { CorePath } from '@app/shared/models';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-give-help',
   templateUrl: './give-help.component.html',
   styleUrls: ['./give-help.component.scss'],
 })
-export class GiveHelpComponent {
+export class GiveHelpComponent implements OnInit {
   corePath = CorePath;
+  editMode = false;
+
+  constructor(private router: Router) {}
+
+  ngOnInit() {
+    this.editMode = this.router.url?.includes(CorePath.Edit);
```

Comment:
nie mamy innej opcji wyciągnięcia tej danej? nie wiem czy czytanie z routera to mocne źródło prawdy

Reply:
nie. router nie jest obligatoryjny. jakieś propozycje?

Reply:
z tego co sprawdziłem, aktualnie i tak się nie da wejść w edycję inaczej niż klikając "Edytuj" (chyba, że wiesz jak?)
https://github.com/coi-gov-pl/gui-web-pomogamukrainie-forum/blob/f4575b764e458af9ff35cb1f33cab921d817c333/src/app/my-account/my-account/my-account.component.ts#L174
jako drugi parametr przyjmuje `extras`, może tam to przekazać?

Reply:
Dla mnie jest ok

Reply:
@klimtron zapinanie się na część URLa, żeby zdecydować czy jesteśmy w `edit mode` czy nie, nie jest najlepszym wyjściem. Takie rzeczy powinny być trzymane w stanie aplikacji. Co jeśli byśmy chcieli się przełączać do `edit mode` bez jakichkolwiek zmian w URL?
Oczywiście nie jest to na ten moment krytyczne i możemy iść z zaproponowanym rozwiązaniem.

@jaroslawkrol widziałem w kodzie w kilku miejscach mamy `this.isEditRoute`
https://github.com/coi-gov-pl/gui-web-pomogamukrainie-forum/blob/d4eabbaee706cf41f65ac8df702e3b7cfff946b8/src/app/give-help/accommodation-form/accommodation-form.component.ts#L42
moglibyśmy użyć tego `this.editMode` zamiast w każdym miejscu mieć osobną implementację?

Reply:
Rozumiem, myślę że jest to do rozważenia, @jaroslawkrol jeśli to nie jest jakiś duży nakład pracy to spróbuj to zrobić po Bożemu. Natomiast isEditRoute nadal polega na url'u